compose/foundation/foundation/src/uikitMain/kotlin/androidx/compose/foundation/DarkTheme.uikit.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.foundation
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.ReadOnlyComposable
+import androidx.compose.ui.uikit.LocalUITraitCollectionState
+import platform.UIKit.UIUserInterfaceStyle.UIUserInterfaceStyleDark
+
+@Composable
+@ReadOnlyComposable
+internal actual fun _isSystemInDarkTheme(): Boolean {
+    return LocalUITraitCollectionState.current.value
+        .userInterfaceStyle == UIUserInterfaceStyleDark
```

Comment:
> [`UIUserInterfaceStyleDark`](https://developer.apple.com/documentation/uikit/uiuserinterfacestyle/uiuserinterfacestyledark) exists since iOS 12

It should be a check for iOS 12 here?

Reply:
Actually adding support for ios 12 wasn't valueable because dark mod was introduces in iOS 13 😅

Reply:
> It should be a check for iOS 12 here?

Compose [won't work](https://github.com/JetBrains/skiko/pull/702) on iOS < 12 because of skia, so this check is not required